include/ozo/connection.h

```diff
@@ -20,6 +20,62 @@ using no_statistics = decltype(hana::make_map());
 template <typename, typename = std::void_t<>>
 struct is_connection : std::false_type {};
 
+template <typename T, typename = std::void_t<>>
+struct get_connection_oid_map_impl {
+    template <typename Conn>
+    static auto apply(Conn&& c) -> decltype(c.oid_map()) { return c.oid_map();}
+    template <typename Conn>
+    static auto apply(Conn&& c) -> decltype((c.oid_map_)) { return c.oid_map_;}
+};
+
+template <typename T>
+inline auto get_connection_oid_map(T&& conn) ->
+    decltype (get_connection_oid_map_impl<std::decay_t<T>>::apply(std::forward<T>(conn))){
```

Comment:
Put `->` on this line and add indent. Now it is hard to distinguish this line from function body.